apps/test/unit/code-studio/pd/professional_learning_landing/landingPageTest.js

```diff
@@ -1,97 +1,66 @@
-import LandingPage, {
-  LastWorkshopSurveyBanner,
-} from '@cdo/apps/code-studio/pd/professional_learning_landing/LandingPage';
-import Button from '@cdo/apps/templates/Button';
+import LandingPage from '@cdo/apps/code-studio/pd/professional_learning_landing/LandingPage';
 
-describe('Tests for Professional Learning Landing Page', () => {
-  const generateLandingPage = (landingPageProps = []) => {
-    return shallow(<LandingPage {...landingPageProps} />);
-  };
+const DEFAULT_PROPS = {
+  lastWorkshopSurveyUrl: 'url',
+  lastWorkshopSurveyCourse: 'CS Fundamentals',
+  deeperLearningCourseData: [{data: 'oh yeah'}],
+  currentYearApplicationId: 2024,
+  workshopsAsParticipant: [{data: 'workshops'}],
+  plCoursesStarted: [{data: 'courses'}],
+};
 
-  describe('Tests related to the initial state of the landing page for given teacher', () => {
-    it('page is as expected for a teacher with a pending survey', () => {
-      const landingPage = generateLandingPage({
-        lastWorkshopSurveyUrl: 'url',
-        lastWorkshopSurveyCourse: 'CS Fundamentals',
-        deeperLearningCourseData: [{data: 'oh yeah'}],
-      });
+describe('LandingPage', () => {
+  function renderDefault(propOverrides = {}) {
+    render(<LandingPage {...DEFAULT_PROPS} {...propOverrides} />);
+  }
 
-      const mainWrapper = landingPage.find('main');
-      expect(mainWrapper.childAt(0).is('LastWorkshopSurveyBanner')).to.be.true;
-      expect(mainWrapper.childAt(0).prop('subHeading')).to.equal(
-        'Submit your feedback'
-      );
-      expect(mainWrapper.childAt(1).is('EnrolledWorkshops')).to.be.true;
-      expect(
-        mainWrapper.find('ProfessionalLearningCourseProgress')
-      ).to.have.length(1);
-    });
-
-    it('page is as expected for a CSD/CSP teacher with a pending survey', () => {
-      const landingPage = generateLandingPage({
-        lastWorkshopSurveyUrl: 'url',
-        lastWorkshopSurveyCourse: 'CS Discoveries',
-        deeperLearningCourseData: [{data: 'oh yeah'}],
-      });
-
-      const mainWrapper = landingPage.find('main');
-      expect(mainWrapper.childAt(0).is('LastWorkshopSurveyBanner')).to.be.true;
-      expect(
-        mainWrapper.childAt(0).shallow().text().indexOf('Submit your feedback')
-      ).to.equal(-1);
-      expect(mainWrapper.childAt(1).is('EnrolledWorkshops')).to.be.true;
-      expect(
-        mainWrapper.find('ProfessionalLearningCourseProgress')
-      ).to.have.length(1);
-    });
-
-    it('page is as expected for a teacher with no pending survey but upcoming workshops and plc enrollments', () => {
-      const landingPage = generateLandingPage({
-        deeperLearningCourseData: [{data: 'oh yeah'}],
-      });
-
-      const mainWrapper = landingPage.find('main');
-      expect(mainWrapper.childAt(0).is('EnrolledWorkshops')).to.be.true;
-      expect(
-        mainWrapper.find('ProfessionalLearningCourseProgress')
-      ).to.have.length(1);
+  it('page is as expected for a new teacher', () => {
```

Comment:
Nit: would you mind making more descriptive test names so future editors will know what is to be expected please? Something like "shows ____ and ___ for a new teacher" in this case

Reply:
Definitely! How do these look: https://github.com/code-dot-org/code-dot-org/pull/57087/commits/a115810a50e2f6bf013696d75a49a2a999482936?